pinc/ProjectTransition.inc

```diff
@@ -982,8 +980,7 @@ function round_to_PP_out_collateral( $old_project, $project, $transition, $who )
         $body_blurb_messages[] = _("This project has completed all rounds and is ready for you to do the post-processing.");
         $body_blurb_messages[] = _("If you do not want to work on it, set it to Available for Post-Processing.");
```

Comment:
I think this should all be one paragraph. I don't mind putting it off for now, but am not quite sure how we should mark this sort of thing for future cleanup.